eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -203,14 +203,15 @@ public void exec() throws IOException {
             new Moja<>(PlaceMojo.class),
         };
         while (true) {
+            final StopWatch times = new StopWatch();
             for (final Moja<?> moja : mojas) {
                 moja.copy(this).execute();
             }
             final String after = this.status();
             ++cycle;
             Logger.info(
-                this, "Assemble cycle #%d (%s -> %s)",
-                cycle, before, after
+                this, "Assemble cycle #%d (%s -> %s), took %[ms]s sec",
```

Comment:
@mximp no need to say `sec` here. `%[ms]s` will be printed as `3s` or `230ms` or `12min`